extensions/wikia/Wall/WallBaseController.class.php

```diff
@@ -49,15 +50,23 @@ public function thread() {
 
 		$id = $this->request->getVal( 'id', null );
 
-		$this->getThread( $id );
+
+		$wallthread = WallThread::newFromId( $id );
+		$wallthread->loadIfCached();
+
+		$threads = [ $id => $wallthread ];
+
+		$this->response->setVal('threads', $threads);
+		$this->response->setVal('title', $this->wg->Title);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)